packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/Model.java

```diff
@@ -1,31 +1,40 @@
 package software.uncharted.terarium.hmiserver.models.dataservice;
 
+import software.uncharted.terarium.hmiserver.models.dataservice.modelparts.ModelMetadata;
+import software.uncharted.terarium.hmiserver.models.dataservice.modelparts.ModelSemantics;
+import software.uncharted.terarium.hmiserver.annotations.TSModel;
+import software.uncharted.terarium.hmiserver.annotations.TSOptional;
+
 import com.fasterxml.jackson.annotation.JsonSetter;
 import com.fasterxml.jackson.annotation.Nulls;
 import lombok.Data;
 import lombok.experimental.Accessors;
 
 import java.io.Serializable;
 import java.time.LocalDateTime;
-import java.util.List;
+import java.util.Map;
+
 @Data
 @Accessors(chain = true)
+@TSModel
 public class Model implements Serializable {
 
-	private Long id;
-
 	private String name;
 
 	@JsonSetter(nulls = Nulls.SKIP)
 	private String description = "";
 
-	private String framework;
+	private String model_version;
 
-	private LocalDateTime timestamp;
+	private String schema;
 
-	private ModelContent content;
+	private Map<String, Object> model;
+
+	private ModelSemantics semantics;
 
-	private Concept concept;
+	private ModelMetadata metadata;
 
-	private List<Object> parameters;
+	// FIXME: deprecated, remove
```

Comment:
Can this be removed?

Reply:
Yep. I will be removing this locally right now. Just need to also change ModelResource to know how to get the new content